### ISSUE TITLE
Pagination key when the button is disabled and pagination text is not helpful for 508 accessibility

### ISSUE THREAD

**Issue by allendol** [appbaseio/reactivesearch#1128]
We are using ReactiveList with pagination. The component should be accessible.
There are two issue with pagination:
1- When the Prev button is disabled the tab will still go there.
2- Number 1, 2,, 3 doesn't help to say what this is. There is no alt tag on href created. Is there any place we can change numbers 1, 2, 3 to like "Page 1", "Page 2", "Page 3", etc?

![Pagination](https://user-images.githubusercontent.com/40170767/63962433-dc1bee80-ca60-11e9-9248-d374aa7c3d75.png)

**Comment by jyash97** [comment]
Thanks, @allendol  I will take a look into this.

**Comment by allendol** [comment]
Hi @jyash97
Thanks for your quick response. I am adding the changes at node_modules\@appbaseio\reactivesearch\lib\components\result\addons\Pagination.js and it doesn't have any affect. It must be somewhere else that I need to update and I don't know where.
